pandas/_libs/tslibs/timezones.pyx

```diff
@@ -42,18 +50,42 @@ cdef tzinfo utc_stdlib = timezone.utc
 cdef tzinfo utc_pytz = UTC
 cdef tzinfo utc_dateutil_str = dateutil_gettz("UTC")  # NB: *not* the same as tzutc()
 
+cdef tzinfo utc_zoneinfo = None
+
 
 # ----------------------------------------------------------------------
 
+cdef inline bint is_utc_zoneinfo(tzinfo tz):
+    # https://github.com/pandas-dev/pandas/pull/46425#discussion_r830633025
+    if tz is None or zoneinfo is None:
+        return False
+
+    global utc_zoneinfo
```

Comment:
shouldn't this be at the module level?

Reply:
this is the workaround [suggested](https://github.com/pandas-dev/pandas/pull/46425#discussion_r830633025) by pganssle for cases with missing tzdata

Reply:
ok can you add a comment to that effect (and ref the issue) as this is a bit non-standard

Reply:
updated + green